modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaBaseModelListenerCheck.testjava

```diff
@@ -7,9 +7,43 @@ package com.liferay.source.formatter.dependencies.upgrade;
 
 public class UpgradeJavaBaseModelListenerCheck extends BaseModelListener<t> {
 
+	@Override
+	public void onAfterCreate(t model) {
+		super.onAfterCreate(model);
+	}
+
+	@Override
+	public void onAfterCreate(t model) {
+		super.onAfterUpdate((t)model.clone(), model);
+
+		super.onAfterUpdate((t)model.clone(), model);
```

Comment:
@kyle-miho is this intended to be doubled up? Same question for `onAfterCreate`.

Reply:
@drewbrokke 
> @drewbrokke fixed, also added an additional method call that needs to be upgrade in 2 of the unit tests with the signature param upgrade and without, to make sure it still works if it needs to cycle through the upgrade more than 1 time

Reply:
its safer because it only cycles through one param upgrade per run, im worried that it might detect or do wrong replaces after the first one, even if it makes the unit test look a bit clunkier if thats okay

Reply:
Ah, thank you, I missed that detail